### PR TITLE
Add rendering of Field of operation pages

### PR DESCRIPTION
**Pull request by JonathanHallam**
This is part of ongoing work to stop Whitehall from rendering anything 

## Whitehall
Screenshot 2023-04-05 at 11 15 46

## Government Frontend
Screenshot 2023-04-05 at 11 19 51

Trello - https://trello.com/c/alzDKosv/472-add-rendering-of-field-of-operation-pages-to-government-frontend

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

File: app/presenters/field_of_operation_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class FieldOfOperationPresenter < ContentItemPresenter
+  include ContentItem::TitleAndContext
+
+  FatalityNotice = Struct.new(:roll_call_introduction, :casualties, :title, :base_path)
```
Comment (BeckaL) [inline app/presenters/field_of_operation_presenter.rb:4]: nice :-)

File: config/locales/en.yml [hunk 2]
```diff
@@ -294,7 +294,11 @@ en:
   fatality_notice:
     alt_text: Ministry of Defence crest
     field_of_operation: Field of operation
+    none_added:
     operations_in: Operations in %{location}
+  field_of_operation:
+    context: British fatalities
+    title_start: Operations in
```
Comment (BeckaL) [inline config/locales/en.yml:301]: Could we change this to being an interpolated value, and rename it to `title`? I think this was how it was done in Whitehall, and is a bit nicer than having half of the text here

File: app/views/content_items/field_of_operation.html.erb [hunk 3]
```diff
@@ -0,0 +1,68 @@
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/title", @content_item.title_and_context %>
+    </div>
+    <div class="govuk-grid-column-one-third govuk-!-padding-top-8 govuk-!-padding-left-0 govuk-!-padding-bottom-8">
+      <%= render "govuk_publishing_components/components/organisation_logo", {
+        organisation: {
+          name: sanitize("Ministry<br>of Defence"),
+          url: "/government/organisations/ministry-of-defence",
```
Comment (BeckaL) [inline app/views/content_items/field_of_operation.html.erb:10]: Could we get this data from the content item's links rather than hard coding it?

File: app/views/content_items/field_of_operation.html.erb [hunk 4]
```diff
@@ -16,11 +16,13 @@
   </div>
 
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-one-third">
-      <%= render "govuk_publishing_components/components/contents_list", {
-        contents: @content_item.contents
-      } %>
-    </div>
+    <% if @content_item.contents.present? %>
```
Comment (BeckaL) [inline app/views/content_items/field_of_operation.html.erb:19]: The commit message is a bit blank for this. Does it make sense to have this as a separate commit, or should we squash into the previous commit?

File: config/locales/ar.yml [hunk 5]
```diff
@@ -590,7 +590,7 @@ ar:
   fatality_notice:
     alt_text: شعار وزارة الدفاع
     field_of_operation: ساحة العمليات
-    none_added:
+    none_added: لم تتم إضافة أي إشعارات حول الوفيات في ساحة العمليات هذه حتى الآن.
```
Comment (BeckaL) [inline config/locales/ar.yml:593]: Could you reword the commit message a bit here to make a bit more sense? I know what it's getting at now I look at the contents of this commit, but was confused on initial reading.

**Review by BeckaL — COMMENTED** [review]
Nice, looks generally good to me, a few suggestions / comments below

**Comment by JonathanHallam** [comment]
> Nice, this is looking really good. Noticed that the logo now renders incorrectly, I wonder if we can specify that we want to use the organisation title rather than the 'formatted title' from the content item Screenshot 2023-04-06 at 10 30 07

We can, but we'd lose the break which we should have, really should have spotted this, sorry, looking for a way I can do both the `html_safe / sanitize` thing that whitehall is currently doing

**Comment by JonathanHallam** [comment]
@BeckaL have fixed this with `html_safe`

File: config/locales/en.yml (hunk 2 above)
Comment (JonathanHallam) [reply config/locales/en.yml]: done :)

File: app/views/content_items/field_of_operation.html.erb (hunk 3 above)
Comment (JonathanHallam) [reply app/views/content_items/field_of_operation.html.erb]: good shout, have amended

File: app/views/content_items/field_of_operation.html.erb (hunk 4 above)
Comment (JonathanHallam) [reply app/views/content_items/field_of_operation.html.erb:14]: have updated

File: config/locales/ar.yml (hunk 5 above)
Comment (JonathanHallam) [reply config/locales/ar.yml:593]: have updated :)

**Review by BeckaL — APPROVED** [review]
Fab, looks good to me, happy for this to go to F&V

**Review by catalinailie — APPROVED** [review]
Looks good to me ⭐

**Comment by chao-xian** [comment]
@JonathanHallam which is the example page from https://government-frontend-pr-2736.herokuapp.com/ ?

**Comment by JonathanHallam** [comment]
@chao-xian https://government-frontend-pr-2736.herokuapp.com/government/fields-of-operation/iraq

Which reminds me that there's the examples in the readme for this app, I'll add a commit to add it there
